api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -482,7 +482,7 @@ function rewardReducer(network, validators, reward, reducers) {
 
 function democracyProposalReducer(network, proposal) {
   return {
-    id: Number(proposal.index),
+    id: `Democracy-`.concat(proposal.index),
```

Comment:
```suggestion
    id: `democracy-`.concat(proposal.index),
```

Reply:
are there problems caching between networks? there could be 2 proposals on different networks with the same id